platform/darwin/src/MGLStyleValue.mm

```diff
@@ -63,6 +63,7 @@ - (instancetype)init {
 
 - (instancetype)initWithBase:(CGFloat)base stops:(NSDictionary *)stops {
     if (self = [super init]) {
+        NSAssert(stops.allKeys.count, @"%@ need at least one stop.", [self class]);
```

Comment:
`[self class]` resolves to `MGLConstantStyleValue` or `MGLStyleFunction`, but I think `self` would be more informative since it calls `-description`.